### ISSUE TITLE
provenance standards

### ISSUE THREAD

**Issue by vjcitn** [HelenaLC/SpatialData#56]
The merfish.zarr in inst/extdata has zmetadata but I do not see anything that would constitute experiment metadata -- what is the source, what version of platform, etc.  Can we state a minimal standard for experiment provenance and ensure that all
our data artifacts can check for compliance?

**Comment by HelenaLC** [comment]
Agreed. To the best of my knowledge, this is currently missing in `SpatialData` (Python). The topic came up with Luca in a recent call. One option would be to stash element-wise metadata (i.e., for every image/label/point/shape/table), which could be accessed in R as a table, and used for easy filtering (e.g., give me all elements from batch A or section 2 etc.). Ideally, we allow for full flexibility here (because any dataset could, in principle, have data from many hospitals, patients, sections, technologies etc.). Minimal entries would simply be the element identifier, I guess.

**Comment by Artur-man** [comment]
This is related to #128. 

There is an experimental implementation that now tracks and validates elements using custom made schema. 
https://github.com/Huber-group-EMBL/SpatialData.validate

However, to this day, a similar schema is still missing from `scverse/spatialdata`.
